applications/cdr/src/cdr_util.erl

```diff
@@ -54,3 +59,16 @@ do_save_cdr(AccountMODb, Doc) ->
 -spec register_views() -> 'ok'.
 register_views() ->
     kz_datamgr:register_views_from_folder('cdr').
+
+-spec check_media_names(list()|kz_term:api_binary()) -> list().
+check_media_names(MediaNames) when is_list(MediaNames) ->
```

Comment:
You can do this all in the function head:

```erlang
check_media_name([HeadMN | TalMN]) ->
```
Implies the `is_list` check, and binds the head and tail of the list, all in one go!